pkgs/tools/misc/coreutils/default.nix

```diff
@@ -10,27 +19,27 @@
 , singleBinary ? "symlinks" # you can also pass "shebangs" or false
 }:
 
-# Note: this package is used for bootstrapping fetchurl, and thus
-# cannot use fetchpatch! All mutable patches (generated by GitHub or
-# cgit) that are needed here should be included directly in Nixpkgs as
-# files.
+# Note: this package is used for bootstrapping fetchurl, and thus cannot use
+# fetchpatch! All mutable patches (generated by GitHub or cgit) that are needed
+# here should be included directly in Nixpkgs as files.
 
 assert aclSupport -> acl != null;
 assert selinuxSupport -> libselinux != null && libsepol != null;
 
-with lib;
-
+let
+  isNotCross = (stdenv.hostPlatform == stdenv.buildPlatform);
```

Comment:
Been meaning to ask this in the previous PR. Why have a negative bool? What about `isCross`? You mostly use it with `!` anyway

Reply:
Yes please, change to isCross or just use the long form. This is hard to read.

Reply:
It looked clearer to me initially.